Recrutify/Recrutify.DataAccess/Repositories/Abstract/ICandidateRepository.cs

```diff
@@ -27,5 +27,9 @@ public interface ICandidateRepository : IBaseRepository<Candidate>
         Task CreateFeedbacksByIdsAsync(IEnumerable<Guid> ids, Guid projectId, Feedback feedback);
 
         Task UpdateStatusByIdsAsync(IEnumerable<Guid> ids, Guid projectId, Status status, string reason);
+
+        Task<List<Location>> GetlocationAsync(Guid? projectId = null);
```

Comment:
Remove = null after param in **ALL** places in PR

Reply:
done